spec/features/views_a_drink_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+feature "User views a drink's details", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.